src/test/unit/specific_feature_tests/native-edit-registration.js

```diff
@@ -0,0 +1,55 @@
+/* global
+    QUnit,
+    prepareUnitTestModule,
+    skipKeyboardShortcutTests,
+    simulateKeyCombo
+*/
+
+"use strict";
+
+module("native-edits-registering", {setup: prepareUnitTestModule});
+
+// Registration of native edits is almost entirely external module code.
+// Specifically, the 'edited' module.
+// The following tests are not meant to fully cover 'edited'.
+// They are meant to test that 'edited' seems to be working.
+
+QUnit.test("edited is instantiated", function () {
+    QUnit.expect(1);
+
+    var wymeditor = jQuery.wymeditors(0),
+        edited = wymeditor.nativeEditRegistration.edited,
+        Edited = WYMeditor.EXTERNAL_MODULES.Edited;
+    QUnit.ok(edited instanceof Edited, "instanceof");
+});
+
+QUnit.test("'edited' seems to be working", function () {
```

Comment:
How about 'edited' modification callbacks are firing on keystrokes.

That's what this is actually testing.